src/data/hashMap/PLHashMap.ts

```diff
@@ -29,7 +29,7 @@ export class PLHashMap<Item extends PLBase> implements PLBase, Index<PLString, I
   }
 
   public toJS(): Map<string, any> {
-    return new Map(Array.from(this._value.entries()).map(([k, v]) => [k, v.toJS()]))
+    return this._value
```

Comment:
why did you change the `toJS` implementation?

Reply:
@maxinteger It wasn't intentional. I reverted the changes.